backend/app/JWT.py

```diff
@@ -0,0 +1,6 @@
+def get_jwt_key():
```

Comment:
So this obviously would not be a key to use in production, I just wanted to test it out to see if it would work. We could pull the key in from another server or something for production since the code would be public on github. Thoughts? Would a key like this do the job?